### ISSUE TITLE
absolute unix paths are mistaken for `BotCommandSuggestion`

### ISSUE THREAD

**Issue by Simon-Laux** [deltachat/message-parser#67]
we could easily solve this by defining in code & spec that bot command suggestions can not contain `/` inside of them (except for the single `/` in the beginning).

CC @adbenitez (because of his expertise with bots and `BotCommandSuggestion`s)

### Example

```
/Users/User/Library/Application 
```

## Expected
```
[
    {
        "t": "Text",
        "c": "/Users/User/Library/Application "
    }
]
```

## Actual

```
[
    {
        "t": "BotCommandSuggestion",
        "c": "/Users/User/Library/Application"
    },
    {
        "t": "Text",
        "c": " "
    }
]
```

**Comment by farooqkz** [comment]
I'm thinking this as invalid. For instance, `/etc` can be considered both a botcommand and a unix absolute path.

**Comment by Simon-Laux** [comment]
> I'm thinking this as invalid.

it is not a question of making everything absolutely "perfect". It's rather a question of how we can make things better step by step.

Means:
`/etc` - parse it as bot command , if people do not want that they can enable markdown mode and write it as `` `\etc` ``
- `/Users/User/Library/Application` contains multiple slashes so I would say it's not likely that it was meant as bot command suggestion.

also let's wait on @adbenitez's opinion

**Comment by adbenitez** [comment]
my opinion is that it is fine to linkify the whole path as command, it can be useful anyway to click it and it is not something terrible, in the past there used to be bots with commands like `/join/group/..`  but that is not common currently, so considering NOT a command if there is more than one `/` would be fine as well

**Comment by farooqkz** [comment]
@Simon-Laux Close it as won't fix?

**Comment by Simon-Laux** [comment]
my opinion is still to do what I proposed. adb said it's not common so a good time to restrict it IMO.

**Comment by farooqkz** [comment]
okay. since my vote is neutral, and I have assigned myself, we'll implement according to your proposed testcase
